plugins_container.txt

```diff
@@ -1 +1 @@
-terraform
\ No newline at end of file
+terraform
```

Comment:
Why do you require the Terraform plugin?

Reply:
Hmm, It's just a integration test for the plugin installation manager. I want to test whether it can work as expected in the workflow.

Reply:
You can add a comment saying so to make it less confusing for the reader